stdlib/public/SDK/CoreImage/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+cmake_minimum_required(VERSION 3.4.3)
+include("../../../../cmake/modules/StandaloneOverlay.cmake")
```

Comment:
All the state should come from a configurable cmake cache file and a top-level target ``CMakeLists.txt``. Modules should add some capability to the system, like how SwiftComponents manages the installed components in an archive the build creates, or how SwiftSource teaches cmake how to compile Swift files, or like FindICU and FindUUID teach cmake how to find libraries.

As for the relative path include problem, I would prefer that cmake has a builtin that recursively finds upwards until a ``.git`` directory is found (or the nth .git directory or a named one, for deeply nested submodules projects, stopping before ``~/`` of course), and call that the actual ``PROJECT_SOURCE_ROOT`` directory. Maybe cmake has this already and I'm not aware. It seems important...

Anyway, this PR is independent of the rest of the build system and the comments within this thread represent the direction I'd like the build system to take. Merging this can't be any worse than what we have now (unless this PR somehow causes build errors that I haven't tested).